code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -239,6 +239,7 @@ var/list/datum/stack_recipe/plastic_recipes = list ( \
 
 var/list/datum/stack_recipe/gold_recipes = list ( \
 	new/datum/stack_recipe("golden floor tile", /obj/item/stack/tile/mineral/gold, 1, 4, 20), \
+	new/datum/stack_recipe("legacy golden floor tile", /obj/item/stack/tile/mineral/gold/gold_old, 1, 4, 20), \
```

Comment:
Rather than "legacy", I'd name it "alternative" or "alternate". Legacy doesn't really fit in-game, even though it is the legacy tile.